README.md

```diff
@@ -8,7 +8,6 @@ Frameworks included in the project:
 
 * [NestJS](https://nestjs.com/) - Backend framework, provides IOC/DI for lifecycle management
 * [Axios](https://axios-http.com/docs/intro) - Promise based HTTP client
-* [Lodash](https://lodash.com/) - Utility framework
```

Comment:
Lodash isn't brought in with the `package.json` which added confusion a couple times when people wanted to do lodash things.